.github/workflows/cibuildwheel.yml

```diff
@@ -27,7 +28,7 @@ jobs:
           CIBW_BUILD: ${{matrix.python}}-*
           CIBW_SKIP: pp* *musllinux*
           # Linux archs
-          CIBW_ARCHS_LINUX: "x86_64"
+          CIBW_ARCHS_LINUX: ${{ fromJSON('["x86_64", "aarch64"]')[matrix.os == 'ubuntu-24.04-arm'] }} # poor man ternary https://github.com/orgs/community/discussions/25725#discussioncomment-3248924
           # MacOS archs
           CIBW_ARCHS_MACOS: "x86_64"
```

Comment:
with the ternary trick we could merge again the arm mac build (mainly for the reason that we have to invent weird names for build_linux_and_macx86